libcxx/test/libcxx/depr/depr.default.allocator/allocator.members/allocate.cxx2a.pass.cpp

```diff
@@ -1,93 +0,0 @@
-//===----------------------------------------------------------------------===//
```

Comment:
I am a bit confused -- why are we removing this test? The name says `allocate.cxx2a.pass.cpp` but it seems that this test was running in all configurations and was our only test for `allocate(size_t, void const*)` in e.g. C++11. Did I miss something?

Reply:
I initially thought that it was checking the allocator members keep working in C++20 when the corresponding macro is defined (and running in other standard modes for sanity checking or simply redundantly as an oversight). I implicitly assumed there are other tests checking these members in pre-C++20 modes.

But I was wrong, thanks for pointing this out, it was the only test that has the deprecated version of allocate with hint, I've restored it and made it run only for pre-C++20 mode.